tests/Behat/FeatureContext.php

```diff
@@ -158,7 +158,38 @@ protected function getNodeByTitle(string $title): NodeInterface {
       throw new \Exception("Multiple nodes with title '$title' found.");
     }
 
-    return reset($nodes);
+    $node = reset($nodes);
+
+    // Load the latest revision.
+    $entity_type = $node->getEntityType();
```

Comment:
Le'ts keep this the way it was and load the latest revision in `theNodeShouldHaveTheFollowingVersion `